deploy/api.yaml

```diff
@@ -291,6 +291,14 @@ components:
         countryCode:
           type: "string"
           example: "GBR"
+#  securitySchemes:
```

Comment:
i assume when we want them to be required we uncomment this code block?

Reply:
Yep that's the idea Matt 👍